Tests/CommandsTests/BuildToolTests.swift

```diff
@@ -27,8 +27,16 @@ final class BuildToolTests: XCTestCase {
         XCTAssert(try execute(["--version"]).contains("Swift Package Manager"))
     }
 
+    func testBinPath() throws {
+        fixture(name: "ValidLayouts/SingleModule/Executable") { path in
+            XCTAssert(try execute(["--bin-path"], packagePath: path).contains("/.build/debug"))
+            XCTAssert(try execute(["-c", "release", "--bin-path"], packagePath: path).contains("/.build/release"))
```

Comment:
I think we should check for the exact path here just to be safe. We can construct that using the `path` variable.